frontend/javascripts/oxalis/model/reducers/reducer_helpers.js

```diff
@@ -1,25 +1,70 @@
 // @flow
 import Maybe from "data.maybe";
 
-import type { APIAnnotation, ServerBoundingBox } from "admin/api_flow_types";
-import type { Annotation, BoundingBoxObject } from "oxalis/store";
+import type { APIAnnotation, ServerBoundingBox, ServerUserBoundingBox } from "admin/api_flow_types";
+import type {
+  Annotation,
+  BoundingBoxObject,
+  UserBoundingBox,
+  UserBoundingBoxToServer,
+} from "oxalis/store";
 import type { Boundary } from "oxalis/model/accessors/dataset_accessor";
 import type { BoundingBoxType } from "oxalis/constants";
 import { V3 } from "libs/mjs";
 import * as Utils from "libs/utils";
 
+function convertServerBoundingBoxToMinMaxBoundingBox(
```

Comment:
`MinMaxBoundingBox` seems to be another term for the same thing that was named `BoundingBox` or `BoundingBoxType` before. It would be great if each of the different bounding box types could have a single name which is used everywhere :)

Reply:
Thanks for finding these inconsistencies. Working on this PR with multiple days in between has its downsides.